src/rules/variableNameRule.ts

```diff
@@ -29,30 +34,48 @@ export class Rule extends Lint.Rules.AbstractRule {
 }
 
 class VariableNameWalker extends Lint.RuleWalker {
+    private checkFormat: boolean;
+    private banKeywords: boolean;
```

Comment:
alphabetize these members. also add the prefix `should` since they're booleans; it's more readable / straightforward (regardless of the fact that they're already of the `boolean` type).